api/db/data.js

```diff
@@ -22,10 +21,12 @@ async function querySuggestions(handle, threshold = 0) {
 
   if (result?.["error"])
     return result;
+
+  console.log(result);
   
   const data = {
     "handle": handle,
-    "suggestions": result.map((e) => {return e["suggestion"];})
+    "suggestions": result
```

Comment:
This will give an object like
```
{
    "handle": "123", 
    "suggestions": [
        {"suggestion": "234", "score": 5},
        {"suggestion": "345", "score": 3}
    ]
}

```